tests/php/test-amp-helper-functions.php

```diff
@@ -467,9 +467,9 @@ public function test_amp_add_amphtml_link_transitional_mode( $data_provider ) {
 		$assert_amphtml_link_present();
 
 		// Make sure adding the filter hides the amphtml link.
-		add_filter( 'amp_frontend_show_canonical', '__return_false' );
+		add_filter( 'amp_frontend_show_amphtml', '__return_false' );
 		$this->assertEmpty( $get_amp_html_link() );
-		remove_filter( 'amp_frontend_show_canonical', '__return_false' );
+		remove_filter( 'amp_frontend_show_amphtml', '__return_false' );
```

Comment:
These can be reverted now, right?

Reply:
Yes, reverted them in 4412fabcf5773ea5f09c91842ab7bd28e9fedfe3.